README.md

```diff
@@ -1,191 +1,182 @@
-Certificate cookbook
-====================
+# Certificate cookbook
 
-[![Build Status](https://secure.travis-ci.org/atomic-penguin/cookbook-certificate.png?branch=master)](http://travis-ci.org/atomic-penguin/cookbook-certificate)
+[![Cookbook Version](https://img.shields.io/cookbook/v/rsync.svg)](https://supermarket.chef.io/cookbooks/certificate)
+[![CI State](https://github.com/sous-chefs/rsync/workflows/ci/badge.svg)](https://github.com/sous-chefs/certificate/actions?query=workflow%3Aci)
+[![OpenCollective](https://opencollective.com/sous-chefs/backers/badge.svg)](#backers)
+[![OpenCollective](https://opencollective.com/sous-chefs/sponsors/badge.svg)](#sponsors)
+[![License](https://img.shields.io/badge/License-Apache%202.0-green.svg)](https://opensource.org/licenses/Apache-2.0)
 
-Description
------------
+## Description
 
-This recipe automates the common task of managing x509 certificates and keys
-from encrypted Data Bags.  This cookbook provides a flexible and re-usable
-LWRP which can be plugged into other recipes, such as the postfix or apache2
+This recipe automates the common task of managing x509 certificates and keys from encrypted Data Bags.  This cookbook
+provides a flexible and re-usable LWRP which can be plugged into other recipes, such as the postfix or apache2
 cookbooks.
 
-#### Warning about Vault mode
+### Warning about Vault mode
 
-Vault mode is not supported in chef-solo, and will result in a failure condition.  One needs
-to select either encrypted, or unencrypted, `data_bag_type` for use with chef-solo.
+Vault mode is not supported in chef-solo, and will result in a failure condition.  One needs to select either encrypted,
+or unencrypted, `data_bag_type` for use with chef-solo.
 
-#### Testing with encrypted data_bags
+### Testing with encrypted data_bags
 
-The KITCHEN.md is a reference document for testing encrypted data_bags with test-kitchen.
-The stub files in`test/integration` are used to validate the `certificate_manage` library.
-These stub files in `test/integration` should not be used in production.  These files include
-self-signed "snake oil" certificate/key and an `encrypted_data_bag_secret` file
-which are not secure to use beyond testing.
+The KITCHEN.md is a reference document for testing encrypted data_bags with test-kitchen. The stub files
+in`test/integration` are used to validate the `certificate_manage` library. These stub files in `test/integration`
+should not be used in production.  These files include self-signed "snake oil" certificate/key and an
+`encrypted_data_bag_secret` file which are not secure to use beyond testing.
```

Comment:
The kitchen.md file was deleted. Should add back kitchen.md or update readme to reflect changes.